### ISSUE TITLE
redirect to portal url instead of database url

### ISSUE THREAD

**Issue by yustas147** [it-projects-llc/odoo-saas-tools#508]
When I click on the link in the letter odoo database sends me, for ex. sale order, and i currently **not logged** in the database, I **redirected to saas portal** database instead of **my client database** (to login). Please explain how to handle this problem. If I already logged in all works fine. 
Thank you!

**Comment by yelizariev** [comment]
Yes, it works this way. After database creation, user has to login in Portal to be redirected to Cliented as authenticated

**Comment by yustas147** [comment]
Thanks for the answer, 
but why it loads portal instead of client database url? Can i change current behaviour? Client should not know about portal, must only know his database url.

**Comment by yelizariev** [comment]
Initially we support only oauth authentication.
I am not sure has it been implemented or not. 
@iledarn, does client databases have password for main (owner) user? Is there a ready solution to have link in a email to client database rather than to portal?

**Comment by iledarn** [comment]
@yelizariev, in saas_portal_signup_custom module we did use the password user gave on signup for his database so he could log in directly. But in general, without using the module owner user may only log in through portal oauth
